pynest/nest/lib/hl_api_spatial.py

```diff
@@ -728,6 +731,81 @@ def GetTargetNodes(sources, tgt_layer, syn_model=None):
     return tuple(src_tgt_map[snode_id] for snode_id in sources.tolist())
 
 
+def GetSourceNodes(src_layer, targets, syn_model=None):
+    """
+    Obtain sources of `targets` in given `source` population.
```

Comment:
Should it be `src_layer` here?
```suggestion
    Obtain sources of `targets` in given `src_layer` population.
```

Reply:
Indeed.